src/components/CustomBulletList/CustomBulletList.tsx

```diff
@@ -15,25 +14,20 @@ const List = styled.ul`
   }
 `;
 
-const StyledTickWhite = styled(TickWhite)`
-  content: '';
-  position: absolute;
-  left: 0;
-  width: 36px;
-  height: 36px;
-`;
-
-const StyledTickBlue = styled(TickBlue)`
-  content: '';
-  position: absolute;
-  left: 0;
-  width: 36px;
-  height: 36px;
+const CheckmarkWrapper = styled.div`
+  color: ${({ color = 'black' }) => color};
```

Comment:
nit: is it necessary? you are not passing prop color to this styled component during render